Formula/swift-format.rb

```diff
@@ -5,7 +5,7 @@ class SwiftFormat < Formula
       tag:      "0.50700.0",
       revision: "3dd9b517b9e9846435aa782d769ef5825e7c2d65"
   license "Apache-2.0"
-  revision 1
+  revision 2
```

Comment:
Oh is there not a 0.50800.0 yet? Revision bump won't be enough since there's a hard coupling of the two (but I elieve this might be fixed from 0.50800.0 onwards).

Reply:
Latest is [0.50700.1](https://github.com/apple/swift-format/releases/tag/0.50700.1) right now.

Reply:
Hopefully it's out in the next day or two otherwise we may have to use a snapshot.

Reply:
I've pulled https://github.com/Homebrew/homebrew-core/labels/CI-long-timeout so the jobs won't run while we work through this.

Reply:
Actually, it looks like it's tagged but not marked as a release: https://github.com/apple/swift-format/releases/tag/swift-5.8-RELEASE

Reply:
Doesn't look like a proper release as `Package.swift` references `branch: main` for its dependencies.

Reply:
`50800.0.0` is out now.

Reply:
Thank you @mrstegeman.

Reply:
@p-linnane the `install` block can probably be removed in the swift-format formula now. It was specific to <= 5.7.

Reply:
Looks like `swift-format` can be built independently now: https://github.com/Homebrew/homebrew-core/pull/127848